Yoast/Yoast-theme-public
# Proper fixing the responsiveness of the black-friday-banner

No more worries about the content, this fix makes sure the purple banner grows or shrinks with the content. If it does not fits in the banner, the content goes to a new line.

**How to test:**
Put lot's of content in the banner and check if it still works
Put just a bit a text in there and check how it works
